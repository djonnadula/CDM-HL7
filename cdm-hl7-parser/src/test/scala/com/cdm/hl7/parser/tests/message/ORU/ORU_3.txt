MSH|^~\&|COCQA1A|COCQA1A|OV|OV|201610251403||ORU^R01|IP_COQ1A_ORU_EDMQC.161102.19|T|2.5||KYA^04446|ER|NE
PID|1||J000426114^^^COCQA1A^MRN|J502719|Test^KM82100||19430920|M||||||||||J00021037132
PV1|1|E|D.ER^^^COCQA1A|EM|||1CSMMW6007^WILLIAMS^NP|||||||||||ER|||||||||||||||||||||||REG
ORC|NEW||J00021037132-VITALS-20161025-1401-1
OBR|1|VITALS|J00021037132-VITALS-20161025-1401-1|VITALS^MIS CLINICAL VITALS|||201610251401|||||||201610251403||1PKLUD7997||QEDMVITALSIGNS
OBX|1|TX|NUR10004^Temperature:||101.9||||||N
OBX|2|TX|NUR10006^Pulse:||31||||||N
OBX|3|TX|NUR10008^Respirations:||31||||||N
OBX|4|TX|NUR10014^Blood Pressure:||131/71||||||N
OBX|5|TX|NUR10018^Ht-Ft:||5||||||N
OBX|6|TX|NUR10018A^Ht-In:||10||||||N
OBX|7|TX|NUR10020^Wt-Lbs:||177||||||N
ZER||201610241023||||||||REG^Registration