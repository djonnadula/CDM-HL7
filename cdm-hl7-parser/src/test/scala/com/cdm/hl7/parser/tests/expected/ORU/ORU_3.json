{
  "0000.COMN": {
    "sending_facility": "COCQA1A",
    "message_control_id": "IP_COQ1A_ORU_EDMQC.161102.19",
    "medical_record_num": "J000426114",
    "medical_record_urn": "J502719",
    "patient_account_num": "J00021037132"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "COCQA1A",
    "sending_facility": "COCQA1A",
    "receiving_application": "OV",
    "receiving_facility": "OV",
    "date_time_of_message": "201610251403",
    "security": null,
    "message_type": {
      "message_code": "ORU",
      "trigger_event": "R01"
    },
    "message_control_id": "IP_COQ1A_ORU_EDMQC.161102.19",
    "processing_id": "T",
    "version_id": "2.5",
    "sequence_num": null,
    "continuation_pointer": {
      "network_id": "KYA",
      "unit_number": "04446"
    },
    "accept_acknowledgement_type": "ER",
    "application_acknowledgement_type": "NE"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id": null,
    "patient_id_list": {
      "id_num": "J000426114",
      "id_check_digit": null,
      "check_digit_scheme": null,
      "assigning_authority": "COCQA1A",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": "J502719",
    "patient_name": {
      "family_name": "Test",
      "given_name": "KM82100"
    },
    "mother_maiden_name": null,
    "date_time_of_birth": "19430920",
    "administrative_sex": "M",
    "patient_alias": null,
    "race": null,
    "patient_address": null,
    "county_code": null,
    "phone_num_home": null,
    "phone_num_business": null,
    "primary_language": null,
    "marital_status": null,
    "religion": null,
    "patient_account_num": "J00021037132"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "E",
    "assigned_patient_location": {
      "point_of_care": "D.ER",
      "room": null,
      "bed": null,
      "facility": "COCQA1A"
    },
    "admission_type": "EM",
    "pre_admit_num": null,
    "prior_patient_location": null,
    "attending_doctor": {
      "person_id": "1CSMMW6007",
      "family_name": "WILLIAMS",
      "given_name": "NP"
    },
    "referring_doctor": null,
    "consulting_doctor": null,
    "hospital_service": null,
    "temporary_location": null,
    "pre_admit_test_indicator": null,
    "readmission_indicator": null,
    "admit_source": null,
    "ambulatory_status": null,
    "vip_indicator": null,
    "admitting_doctor": null,
    "patient_type": "ER",
    "visit_num": null,
    "financial_class": null,
    "charge_price_indicator": null,
    "courtesy_code": null,
    "credit_rating": null,
    "contract_code": null,
    "contract_effective_date": null,
    "contract_amt": null,
    "contract_period": null,
    "interest_code": null,
    "transfer_to_bad_debt_code": null,
    "transfer_to_bad_debt_date": null,
    "bad_debt_agency_code": null,
    "bad_debt_transfer_amt": null,
    "bad_debt_recovery_amt": null,
    "delete_account_indicator": null,
    "delete_account_date": null,
    "discharge_disposition": null,
    "discharged_to_location": null,
    "diet_type": null,
    "servicing_facility": null,
    "bed_status": null,
    "account_status": "REG"
  },
  "0004.ORC": {
    "order_control": "NEW",
    "placer_order_num": null,
    "filler_order_num": "J00021037132-VITALS-20161025-1401-1"
  },
  "0005.OBR": {
    "set_id": "1",
    "placer_order_num": "VITALS",
    "filler_order_num": "J00021037132-VITALS-20161025-1401-1",
    "universal_service_id": {
      "id": "VITALS",
      "text": "MIS CLINICAL VITALS"
    },
    "priority": null,
    "requested_date_time": null,
    "obsv_date_time_num": "201610251401",
    "obsv_end_date_time_num": null,
    "collection_volume": null,
    "collector_id": null,
    "specimen_action_code": null,
    "danger_code": null,
    "relevant_clinical_information": null,
    "specimen_received_date_time": "201610251403",
    "specimen_source": null,
    "ordering_provider": "1PKLUD7997",
    "order_callback_phone_num": null,
    "placer_field_1": "QEDMVITALSIGNS"
  },
  "0006.OBX": {
    "set_id": "1",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10004",
      "text": "Temperature:"
    },
    "obsv_sub_id": null,
    "obsv_value": "101.9",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "N"
  },
  "0007.OBX": {
    "set_id": "2",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10006",
      "text": "Pulse:"
    },
    "obsv_sub_id": null,
    "obsv_value": "31",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "N"
  },
  "0008.OBX": {
    "set_id": "3",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10008",
      "text": "Respirations:"
    },
    "obsv_sub_id": null,
    "obsv_value": "31",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "N"
  },
  "0009.OBX": {
    "set_id": "4",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10014",
      "text": "Blood Pressure:"
    },
    "obsv_sub_id": null,
    "obsv_value": "131/71",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "N"
  },
  "0010.OBX": {
    "set_id": "5",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10018",
      "text": "Ht-Ft:"
    },
    "obsv_sub_id": null,
    "obsv_value": "5",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "N"
  },
  "0011.OBX": {
    "set_id": "6",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10018A",
      "text": "Ht-In:"
    },
    "obsv_sub_id": null,
    "obsv_value": "10",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "N"
  },
  "0012.OBX": {
    "set_id": "7",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10020",
      "text": "Wt-Lbs:"
    },
    "obsv_sub_id": null,
    "obsv_value": "177",
    "units": null,
    "references_range": null,
    "interpretation_codes": null,
    "probability": null,
    "nature_of_abnormal_test": null,
    "obsv_result_status": "N"
  },
  "0013.ZER": {
    "expected_arrival_date_time": null,
    "arrival_date_time": "201610241023",
    "triage_date_time": null,
    "mpi_search": null,
    "complaint": null,
    "priority": null,
    "location": null,
    "room_treatment": null,
    "wait_list": null,
    "current_status_event": {
      "event_code": "REG",
      "event_desc": "Registration"
    }
  }
}